charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```
@@ -23,6 +23,7 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/service-account-signing-key: {{ include ( print $.Template.BasePath "/service-account-signing-key-secret.yaml") . | sha256sum }}
```

```suggestion
        {{- if .Values.serviceAccountConfig }}
        {{- if .Values.serviceAccountConfig.signingKey }}
        checksum/service-account-signing-key: {{ include ( print $.Template.BasePath "/service-account-signing-key-secret.yaml") . | sha256sum }}
        {{- end }}
        {{- end }}        
```

Yes, it would also works without the `if`, but then we might restart the kube-apiserver unnecessarily.